pkgs/stdenv/linux/make-bootstrap-tools-cross.nix

```diff
@@ -179,7 +179,7 @@ rec {
         for i in as ld ar ranlib nm strip readelf objdump; do
           cp ${binutils.out}/bin/$i $out/bin
         done
-        cp -d ${binutils.lib}/lib/lib*.so* $out/lib
+        cp -d ${binutils.out}/lib/lib*.so* $out/lib
```

Comment:
What libraries are left in `${binutils.out}/lib/`?

Reply:
None. That line could go actually.

Reply:
It should, as the `lib/` itself is gone from binutils.